### PR TITLE
Hapi19

### PR DESCRIPTION
**Pull request by pmcclory-contrast**

### CODE REVIEW

File: hapi19/server.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+const { PORT = 3000, HOST = 'localhost', SSL } = process.env;
+const isHttps = SSL === 1;
+
+const manifest = {
+  server: {
+    debug: { request: ['error', 'uncaught'] },
+    port: PORT,
+    host: HOST,
+    routes: {
```
Comment (pmcclory-contrast) [inline hapi19/server.js]: this right here is the only difference between the hapi18 and hapi19 apps - they switched to multipart support being off by default - https://github.com/hapijs/hapi/issues/3920

Comment (bizob2828) [reply hapi19/server.js]: this is weird, no seeing why they turned this off but good catch

File: hapi19/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      path: '/',
+      handler: {
+        view: 'index'
+      }
+    },
+    {
+      method: 'GET',
+      path: '/public/app.css',
+      handler(request, h) {
+        return h.file('public/app.css');
```
Comment (bizob2828) [inline hapi19/routes/index.js]: I fixed this in master so you apps work when starting in root of NodeTestBenches

```suggestion
        return h.file(`${__dirname}/public/app.css`);
```

File: .gitignore [hunk 3]
```diff
@@ -1,10 +1,9 @@
 .vscode/
 node_modules/
 # form upload paths
-loopback/server/uploads/*
```
Comment (bizob2828) [inline .gitignore]: we still need this

Comment (pmcclory-contrast) [reply .gitignore]: whoops rushed that.  would be nice if we were using consistent paths for these....

File: hapi19/routes/index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      path: '/',
+      handler: {
+        view: 'index'
+      }
+    },
+    {
+      method: 'GET',
+      path: '/public/app.css',
+      handler(request, h) {
+        h.file(`${__dirname}/public/app.css`);
```
Comment (bizob2828) [inline hapi19/routes/index.js]: this path is wrong. i needs to be 
```
h.file(`${__dirname}/../public/app.css);
```

**Pull request merged**